src/Microsoft.AspNetCore.DataProtection.Extensions/DataProtectionProvider.cs

```diff
@@ -40,22 +50,83 @@ public static IDataProtectionProvider Create(DirectoryInfo keyDirectory)
             {
                 throw new ArgumentNullException(nameof(keyDirectory));
             }
+            if (setupAction == null)
+            {
+                throw new ArgumentNullException(nameof(setupAction));
+            }
+
+            return CreateProvider(keyDirectory, setupAction, certificate: null);
+        }
 
+#if !NETSTANDARD1_3 // [[ISSUE60]] Remove this #ifdef when Core CLR gets support for EncryptedXml
+        /// <summary>
+        /// Creates an <see cref="DataProtectionProvider"/> given a location at which to store keys
+        /// and a <see cref="X509Certificate2"/> for encryption.
```

Comment:
I'm probably nit-picking, but I'd update this comment a bit to indicate that the certificate is used to encrypt the keys, not the protected data.